### ISSUE TITLE
Ace Jumper Activated

### ISSUE THREAD

**Issue by syntag** [lucax88x/CodeAceJumper#27]
When you activate Ace Jumper, would be nice to have a more visual activated state (as someone who likes to have their status bar visibility off).

I played around with the idea of adding a decorator to overlay the entire VS Code window and adding the following styles:
```css
.overlay {
    position: fixed;
    width: 100%;
    height: 100%;
    z-index: 9999;
    background-color: #ffffff;
    mix-blend-mode: hue;
    top: 0;
    right: 0;
    bottom: 0;
    left: 0;
    pointer-events: none;
}
```

The code above will overlay the window and add a black-and-white effect to it. Would also be nice if the opacity of the black and white effect could be configurable by the user.

**Comment by lucax88x** [comment]
Believe it or now I was trying the same yesterday evening, trying to add a decorator for that.

Asap I deployed the new 2.0.0 I will think about this.

**Comment by lucax88x** [comment]
@SYNTAG how do you add a overlay with that style? you have a snippet?

**Comment by syntag** [comment]
> Believe it or now I was trying the same yesterday evening, trying to add a decorator for that.
> 
> Asap I deployed the new 2.0.0 I will think about this.

That's excellent news!

> @SYNTAG how do you add a overlay with that style? you have a snippet?

I don't have a snippet, but I did test the CSS code above on one of your existing decorators and it worked.

**Comment by lucax88x** [comment]
Did you put that in the code or using the devtools?

**Comment by syntag** [comment]
I did it in DevTools, here's a CodePen I made to give you a visual. I think you'll need to use `position: fixed;` for the actual decorator instead of `position: absolute;` as I did on the CodePen.

https://codepen.io/SYNTAG/pen/XGvapB

Another alternative would be to add a glow effect like this:
https://codepen.io/SYNTAG/pen/YgmEpj

And finally, maybe a cleaner option would be to add a styled decorator at the cursor selection only when Ace Jumper is activated.

**Comment by lucax88x** [comment]
Yeah, but we cannot add custom styles to decorators :( that's the problem.

**Comment by lucax88x** [comment]
@SYNTAG luckly I found an another way to dim the editor, I'll tune, make it configurable and release it for 2.1.0

**Comment by lucax88x** [comment]
released dim in 2.1.0

**Comment by syntag** [comment]
@lucax88x Glad you found a way! Played around with the update and I love it. Good job.
